### PR TITLE
Fixed styling in layout and header

### PR DESCRIPTION
**Pull request by nathanajah**
- Background colors fixed
- Page not extending to the bottom of the screen fixed

### CODE REVIEW

**Comment by kevinluvian** [comment]
buat breadcrumbsDiv mending dikasih padding atau pake <Divider hidden /> aja ya?

**Comment by vandeput** [comment]
I think the way of thinking it is if we have various theme, will all of them have Divider For something below breadcrumbs? Even that `hidden` property there is debatable.

**Comment by nathanajah** [comment]
@vandeput: Can you elaborate more? Not sure if I get what you mean by "If we have various theme.." and "the `hidden` property is debatable".

**Comment by vandeput** [comment]
@nathanajah let me rephrase: is the divider aesthetic or semantic?

**Comment by nathanajah** [comment]
I see. I think in this case, the divider would be aesthetic.

**Comment by vandeput** [comment]
Then perhaps a TODO comment there is good enough.

**Comment by vandeput** [comment]
Ups, just realized that @nathanajah already done the correct way. I am still too sleepy in morning :laughing:
